assembl/static2/.eslintrc.json

```diff
@@ -31,7 +31,7 @@
     "no-mixed-operators": 0,
     "no-console": 2,
     "func-names": ["warn", "never"],
-    "arrow-body-style": ["error", "always"],
+    "arrow-body-style": ["error", "as-needed", { "requireReturnForObjectLiteral": true }],
```

Comment:
so, we don't want this form : 

```() => ({ foo: 'foo' })``` ?

(I understand it can be "strange" and maybe difficult to read but I got used to it (but I'm ok with this rule since prettier allow me to write it like I want ;-) ))

Reply:
Yes. We can discuss it of course. I don't really have a preference. There may be less code changes though, that's it.